### PR TITLE
Dp24 pretextgraph

### PR DESCRIPTION
**Pull request by DLBPointon**
Pretext Ingestion!

The accessory files are now ingested into the pretext file, rather than needing the end user to ingest them independently.
coverage, log2 coverage and repeat density are ingested by default and telomere + gap are ingested if they contain > 0 bytes. 

This highlighted a small bug with the new log_coverage track, which has been corrected with a `round(value, 2)` ensuring that there will always be a whole number for ingestion and no data is skipped.

### CODE REVIEW

**Comment by DLBPointon** [comment]
Pipeline fails as it needs Pretext_hires to be emitted from hic_mapping but can't as pretext_hires needs 16GB minimum

**Comment by DLBPointon** [comment]
This PR has grown a bit, so to summarise:
- Multiple fixes to longreadlogcoverage
- Adding logcoverage to pretextgraph
- Adding pretextgraph
- Updating and fixing docs (will become clear once added to dev)
- Stopped exporting pre.bed
- Start exporting another repeat coverage file for use in JBrowse2

File: docs/images/v1-1-0/treeval_1_1_0_generate_genome.png
Comment (weaglesBio) [inline docs/images/v1-1-0/treeval_1_1_0_generate_genome.png:1]: 1. Terminator is not used in other images, is it needed or clear what it is doing.
2. Fasta label is capitalised, other examples are not.

File: nextflow.config
Comment (weaglesBio) [inline nextflow.config:1]: Does cleanup = true need to be added locally, or will it be globally added to the sanger profile?

**Review by weaglesBio — CHANGES_REQUESTED** [review]

File: docs/images/v1-1-0/treeval_1_1_0_generate_genome.png
Comment (DLBPointon) [reply docs/images/v1-1-0/treeval_1_1_0_generate_genome.png:1]: Fixed

File: nextflow.config
Comment (DLBPointon) [reply nextflow.config:1]: I've reverted this so, cleanup is not back as a profile. I just forgot what I wrote does.
To use, do `-profile singularity,sanger,cleanup`

This way it is optional and global.

**Review by weaglesBio — APPROVED** [review]
Same Fasta issue on gap_finder, insilico_digest (with Genome also) and the full diagram. This is not a blocker though.

**Pull request merged**